packages/happo-target-firefox/src/waitForImagesToRender.js

```diff
@@ -1,7 +1,13 @@
 import findBackgroundImageUrls from './findBackgroundImageUrls';
 
-function waitForImageToLoad(url) {
+export function waitForImageToLoad(url) {
   return new Promise((resolve, reject) => {
+    // No need to wait for the image to be loaded.
+    if (url.indexOf('data:image') === 0) {
```

Comment:
Ah interesting.

>  Are you saying that even with not network latency, it's preferable to load the image first?

Yes, because image rendering is non-instant and non-blocking, so screenshots can actually be taken in the middle of image rendering, which makes it so you end up with a lot of visual diffs for components that use images because the screenshots are taken at different moments while the image was rendering. Here's some more info: https://github.com/Galooshi/happo/issues/162#issuecomment-251465911